Core/Src/porting/common.c

```diff
@@ -67,3 +69,206 @@ void odroid_audio_mute(bool mute)
 
     audio_mute = mute;
 }
+
+common_emu_state_t common_emu_state = {
+    .frame_time_10us = 100000 / 60,  // Most (all?) emus are 60fps
```

Comment:
NES in PAL mode runs at 50 FPS

Reply:
it gets set to that in `main_nes.c`, this is just a reasonable default